testnets/xrplevmtestnet/chain.json

```diff
@@ -0,0 +1,413 @@
+{
+    "$schema": "../../chain.schema.json",
+    "chain_name": "xrplevmtestnet",
+    "status": "live",
+    "network_type": "testnet",
+    "website": "https://xrplevm.org/",
+    "pretty_name": "XRPL EVM Testnet",
+    "chain_type": "cosmos",
+    "chain_id": "xrplevm_1449000-1",
+    "bech32_prefix": "ethm",
+    "node_home": "~/.exrpd",
+    "daemon_name": "exrpd",
+    "key_algos": [
+    "ethsecp256k1"
+    ],
+    "extra_codecs": [
+        "ethermint"
+    ],
+    "slip44": 60,
+    "fees": {
+      "fee_tokens": [
+        {
+          "denom": "axrp",
+          "fixed_min_gas_price": 250000000,
+          "low_gas_price": 200000000000,
+          "average_gas_price": 250000000000,
+          "high_gas_price": 400000000000   
+        }
+      ]
+    },
+    "codebase": {
+      "git_repo": "https://github.com/xrplevm/node",
+      "recommended_version": "v7.0.0",
+      "compatible_versions": [
+        "v7.0.0"
+      ],
+      "consensus": {
+        "type": "cometbft",
+        "version": "v0.38.17"
+      },
+      "binaries": {
+        "linux/amd64": "https://github.com/xrplevm/node/releases/download/v7.0.0/node_7.0.0_Linux_amd64.tar.gz",
```

Comment:
These binaries don't match. In chain.json it has node_7.0.0 while in versions it has exrpd_7.0.0. Which is correct? Once they match I believe the checks will pass.